zfit/models/dist_tfp.py

```diff
@@ -493,3 +493,60 @@ class PoissonPDFRepr(BasePDFRepr):
     hs3_type: Literal["Poisson"] = Field("Poisson", alias="type")
     x: SpaceRepr
     lam: Serializer.types.ParamTypeDiscriminated
+
+
+class ChiSquared(WrapDistribution, SerializableMixin):
+    _N_OBS = 1
+
+    def __init__(
+        self,
+        ndof: ztyping.ParamTypeInput,
+        obs: ztyping.ObsTypeInput,
+        *,
+        extended: ExtendedInputType = None,
+        norm: NormInputType = None,
+        name: str = "ChiSquared",
+    ):
+        """ChiSquared distribution for a ndof degrees of freedom.
+
+        The chisquared shape for $d$ degrees of freedom is defined as
+
+        .. math::
```

Comment:
An empty line is needed between the directive and the content (yes, RST is quite sensitive to many things: https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-math)